concepts/05 UI Components/TreeList/10 Columns/50 Adaptability/05 Hide Columns.md

```diff
@@ -0,0 +1,97 @@
+To enable column hiding, set the [columnHidingEnabled](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/#columnHidingEnabled) property to **true**.
```

Comment:
```suggestion
When cell text overflows the cell, the TreeList truncates the text or adds horizontal scrolling. As an alternative, the component can hide one or several columns to make space for the overflowing text. To enable this feature, set the [columnHidingEnabled](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/#columnHidingEnabled) property to **true**.
```

Reply:
It is necessary to add:
When cell text overflows the cell **or all the columns do not fit the UI component's width**, the TreeList truncates the text or adds horizontal scrolling, **respectively**. As an alternative, the component can hide one or several columns to make space for the overflowing text. To enable this feature, set the [columnHidingEnabled](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/#columnHidingEnabled) property to **true**.

Reply:
But these are one and the same. Columns do not fit, hence cell texts do not fit. You can say:

> When the total width of columns exceeds the component's width, the TreeList truncates column cell texts or adds horizontal scrolling.

Reply:
The situation when the columns don't fit doesn't imply that cell texts don't fit: https://codepen.io/PavelDX/pen/XWprMYQ

Reply:
```suggestion
When the total width of columns exceeds the component's width, the TreeList truncates column cell texts or adds horizontal scrolling. As an alternative, the component can hide one or several columns to make space for the overflowing text. To enable this feature, set the [columnHidingEnabled](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/#columnHidingEnabled) property to **true**.
```